OpenNefia.Content/CharaMake/CharaMakeAttributeRerollLayer.cs

```diff
@@ -0,0 +1,239 @@
+﻿using OpenNefia.Content.Prototypes;
+using OpenNefia.Content.Skills;
+using OpenNefia.Content.UI;
+using OpenNefia.Content.UI.Element;
+using OpenNefia.Content.UI.Element.Containers;
+using OpenNefia.Content.UI.Element.List;
+using OpenNefia.Core;
+using OpenNefia.Core.Audio;
+using OpenNefia.Core.Locale;
+using OpenNefia.Core.Prototypes;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace OpenNefia.Content.CharaMake
+{
+    [Localize("Elona.CharaMake.AttributeReroll")]
+    public class CharaMakeAttributeRerollLayer : CharaMakeLayer
+    {
+        public enum AttributeRerollChoice
+        {
+            Reroll,
+            Proceed
+        }
+        public class AttributeRerollData
+        {
+            public object Data;
```

Comment:
Could make this into an abstract record type and `switch` on that. See `UIResult` for an example.

Reply:
done in c06412c